common/src/address.rs

```diff
@@ -165,6 +165,9 @@ const GZ_ADDRESS_INDEX: usize = 2;
 /// The maximum number of addresses per sled reserved for RSS.
 pub const RSS_RESERVED_ADDRESSES: u16 = 32;
 
+// The maximum number of addresses per sled reserved for control plane services.
+pub const CP_SERVICES_RESERVED_ADDRESSES: u16 = 0xFFFF;
```

Comment:
What about instances which have been allocated below this threshold that are currently deployed on systems?

Is it an issue that they exist within this reserved range? I don't think *future* allocations will land there, but just trying to ensure we don't make assumptions about prior ones.

Reply:
I asked about this in dogfood chat and got this response from @bnaecker:
> It does look like the Propolis server IP address is assigned on instance start, not creation.

Since we currently perform schema upgrades with shutdown instances this shouldn't be a problem. It should probably be tested on real hardware with real instances.

Reply:
That's based on this, for reference:

https://github.com/oxidecomputer/omicron/blob/9eba46df9980aa76a5043cf0300feddc6e79202c/nexus/src/app/sagas/instance_start.rs#L153-L163

Reply:
> That's based on this, for reference:
> 
> https://github.com/oxidecomputer/omicron/blob/9eba46df9980aa76a5043cf0300feddc6e79202c/nexus/src/app/sagas/instance_start.rs#L153-L163

Ha, I just dug that up and was about to put that here :) Thanks!

Reply:
Gotcha, that makes sense. And later, when we do migrations, we'd allocate a new instance IP anyway, so this all checks out to me.